src/main/bash/sdkman-main.sh

```diff
@@ -44,8 +44,21 @@ function sdk {
 			COMMAND="default";;
 		b)
 			COMMAND="broadcast";;
+		ho)
+			COMMAND="home";;
 	esac
 
+	#
+	# Print home directory - special case
+	# Printing the home directory does not require reaching out to the internet.
+	# Printing the home directory also has the hard requirment of not printing
+	# anything else unless it is an actual error
+	#
```

Comment:
Let's drop comments from the source and keep it inside the feature file.

Reply:
done